src/NServiceBus.Core/Persistence/PersistenceDefinition.cs

```diff
@@ -68,6 +68,14 @@ public bool HasSupportFor(Storage storage)
             return storageToActionMap.ContainsKey(typeof(T));
         }
 
+        /// <summary>
+        /// True if supplied storage is supported
+        /// </summary>
+        public bool HasSupportFor(Type storageType) 
```

Comment:
@johnsimons there already is a public one that is generic so I don't see adding a non-generic complement as widening our API surface, just making the API more complete